src/Microsoft.Diagnostics.Monitoring.RestServer/IUrlPortsProvider.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Diagnostics.Monitoring.RestServer
+{
+    /// <summary>
+    /// Provides port information about the URLs the were successfully bound.
+    /// </summary>
+    internal interface IUrlPortsProvider
```

Comment:
Not happy with the name; open to suggestions (and for anything else). Maybe rename to IBoundPortsProvider?

Reply:
Given the functionality, you may want to call it something like IMetricPortsProvider.